LondonBritishLibrary/orient/BLorient754.xml

```diff
@@ -198,8 +199,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                  <persName ref="PRS6905Matthew">አባ፡ ማቴዎስ፡</persName> ወብፁዕ፡ ጳጳስነ፡ <persName ref="PRS6776Marqos">አባ፡
                 ማርቆስ፡</persName> ስብሐት፡ ለእግዚአብሔር፡ <gap reason="ellipsis"></gap> ዝንቱ፡ መጽሐፍ፡ ዘአካለ፡ ክርስቶስ፡
               <gap reason="ellipsis"></gap> ዘአጥርዮ፡ በንዋዩ፡ ከመ፡ ይኩኖ፡ ለመድሕሓኒተ፡ ነፍስ፡ ወሥጋ፡</seg>
-              <note>The colophon has been tampered with, especially in its latter part. The part of the
-              manuscript seems to have been written somewhat earlier than the former part, that is
+              <note>The colophon has been tampered with, especially in its latter part.</note>
+                <note>The part of the
+              manuscript, namely <ref target="#p1_i7.3 #p1_i7.4 #p1_i7.4 #p1_i8 #p1_i9 #p1_i10"></ref> seems
```

Comment:
I still wonder whether there should be 3 msParts in this MS

Reply:
#p1_i7.4 is listed twice

Reply:
I think is should rather be encoded as described here: https://betamasaheft.eu/Guidelines/?q=production&id=syntaxe
as different production Units, but not as msParts

Reply:
I mean the section on texts added in a later hand

Reply:
But the problem is, I can't really undertsand the limits of these production Units, it is not really clear from the catalogue description, because Wright refers often to "this part of the manuscript" which is very ambiguous.